src/routes/(admin)/account/api/+page.server.ts

```diff
@@ -209,6 +209,49 @@ export const actions = {
       redirect(303, "/login/current_password_error")
     }
 
+    const { data: customer, error: cusError } = await supabaseServiceRole
+      .from("stripe_customers")
+      .select("stripe_customer_id")
+      .eq("user_id", user.id)
+      .single()
+
+    if (cusError) {
+      console.error("Error getting stripe customer ID.", cusError)
+      return fail(500, {
+        errorMessage: "Unknown error. If this persists please contact us.",
+        currentPassword,
```

Comment:
definitely shouldn't be returning their password in an error message in plaintext